### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MKanski1**
Added line 3 to the poem.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Poem
 This is an open source poem. All contributions would be great!
+The poem is really well made.
```
Comment (JanSawicki) [inline README.md:3]: Very nice rhyme 😄

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
 Very good poem, I like it, mate!
+The poem is well made.
```
Comment (JanSawicki) [inline README.md:5]: I think it would be a good idea to **come up with a more interesting word** for the rhyme, 
e.g.:
- paid, 
- laid, 
- fade.

**Review by JanSawicki — APPROVED** [review]
Very nice PR, just a tiny nitpick - thank you for your contribution 🎂

File: README.md (hunk 2 above)
Comment (MKanski1) [reply README.md:5]: Thanks for comments 👍

**Pull request closed without merging**